pyuvdata/uvdata/aipy_extracts.py

```diff
@@ -15,20 +15,11 @@
 
 try:
     from pyuvdata import _miriad
-except ImportError:
-    # See setup.py for an explanation. The workaround here is kind of annoying
-    # since the code below needs the name `_miriad.UV` to be a class in order
-    # to import successfully.
-    import os
-
-    if os.environ.get("PYUVDATA_IGNORE_EXTMOD_IMPORT_FAIL", "") != "1":
-        raise
-
-    class UV(object):
-        pass
-
-    _miriad = UV()  # eww gross but it works
-    _miriad.UV = UV
```

Comment:
we don't actually need this silly catch-reassigning anymore since we don't have that weird "need pyuvdata installed to install pyuvdata" _feature_ anymore right? Just concerned about removing this work around if it is still needed.

Reply:
Yeah, before we did `import pyuvdata.version` in the setup.py to generate versioning info on install, but now we do it in a saner way. I'm pretty sure we can get rid of this now. (Empirically installation seems to work without it...)